datarobot_batch_scoring/batch_scoring.py

```diff
@@ -772,7 +807,8 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                           out_file, keep_cols, delimiter,
                           dataset, pred_name,
                           timeout, ui, fast_mode, auto_sample,
-                          dry_run, encoding, skip_dialect):
+                          dry_run, encoding, skip_dialect,
+                          max_batch_size=3.5 * 1024 ** 2):
```

Comment:
We have assumption on nearly equivalent size of each rows in autosampler, but looks like in some cases this assumption is broken when we have some not so regular datasets.

So it was implied that this code needs to check each chunk to ensure we are not hitting limits.